### PR TITLE
fix(Trigger): afterPopupVisibleChange closure save the last visible v…

### PR DESCRIPTION
**Pull request by earlymeme**
…alue  when popupvisible is false

First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)
当popupVisible变成false后，afterPopupVisibleChange闭包里保存的val是上一次visible为true的时候保存的状态
> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/vc-trigger/Trigger.jsx [hunk 1]
```diff
@@ -97,10 +97,10 @@ export default {
         this.sPopupVisible = val;
       }
     },
-    sPopupVisible(val) {
+    sPopupVisible() {
       this.$nextTick(() => {
         this.renderComponent(null, () => {
-          this.afterPopupVisibleChange(val);
+          this.afterPopupVisibleChange(this.popupVisible);
```
Comment (tangjinzhou) [inline components/vc-trigger/Trigger.jsx]: this.sPopupVisible  ?

Comment (earlymeme) [reply components/vc-trigger/Trigger.jsx]: this.popupVisible always equal this.sPopupVisible

Comment (tangjinzhou) [reply components/vc-trigger/Trigger.jsx]: popupVisible 为 false 之后， sPopupVisible也会更改为false，依然还是会调用afterPopupVisibleChange的呀   有没有相关issue  我现在没想到会有什么问题

Comment (earlymeme) [reply components/vc-trigger/Trigger.jsx]: https://codesandbox.io/s/vue-antd-template-o3sxq
当tooltip隐藏的时候，执行afterPopupVisibleChange，这个时候的val的值是true，应该是false才对

Comment (tangjinzhou) [reply components/vc-trigger/Trigger.jsx]: 诡异   这里改成 `this.afterPopupVisibleChange(this.sPopupVisible);` 吧
`this.sPopupVisible` 和 `this.popupVisible` 并不是恒等的

**Pull request closed without merging**